sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java

```diff
@@ -12,6 +12,13 @@
 public final class TestEnvironment {
     private static final ClientLogger LOGGER = new ClientLogger(TestEnvironment.class);
 
+    private static final String scheme;
+
+    static {
+        String envScheme = Configuration.getGlobalConfiguration().get("AZURE_TEST_SCHEME");
```

Comment:
I'd call it AZURE_STORAGE_TEST_SCHEME or so. It's not a repo wide flag.

Consider making it a boolean, i.e. AZURE_STORAGE_TEST_DISABLE_HTTPS. there are only two options here.